FinSearchUnified/BusinessLogic/Models/FindologicArticleModel.php

```diff
@@ -715,8 +720,12 @@ protected function setProperties()
             $allProperties[] = new Property('release_date', ['' => $releaseDate]);
         }
 
-        /** @var \Shopware\Models\Attribute\Article $attributes */
-        $attributes = $this->baseArticle->getAttribute();
+        if (is_callable([$this->baseArticle, 'getAttribute']) && !is_null($this->baseArticle->getAttribute())) {
```

Comment:
* Why the additional check for `is_null`?
* The requested comment explaining why this check has to be done, is missing.

Reply:
Based on my previous comment above, the getAttribute in the Article is always returning null as it assigns it to the mainDetail, and hence the `attr1` is never set in the Article attribute, but always in the Detail.